test-files/io/ServiceHootApiDbBulkInserterTest/psqlOffline.osm

```diff
@@ -27,34 +27,35 @@
         <tag k="area" v="yes"/>
         <tag k="name" v="w1"/>
     </way>
-    <way visible="true" id="2" timestamp="2017-09-21T14:54:50Z" version="1" changeset="23">
+    <way visible="true" id="2" timestamp="2017-11-29T21:11:52Z" version="1" changeset="7">
         <nd ref="5"/>
         <nd ref="6"/>
         <tag k="name" v="w2"/>
         <tag k="highway" v="track"/>
     </way>
-    <way visible="true" id="3" timestamp="2017-09-21T14:54:50Z" version="1" changeset="23">
+    <way visible="true" id="3" timestamp="2017-11-29T21:11:52Z" version="1" changeset="7">
         <nd ref="7"/>
         <nd ref="8"/>
         <tag k="name" v="w3"/>
         <tag k="highway" v="road"/>
     </way>
-    <way visible="true" id="4" timestamp="2017-09-21T14:54:50Z" version="1" changeset="23">
+    <way visible="true" id="4" timestamp="2017-11-29T21:11:52Z" version="1" changeset="7">
         <nd ref="9"/>
         <nd ref="10"/>
         <nd ref="11"/>
         <nd ref="9"/>
     </way>
-    <way visible="true" id="5" timestamp="2017-09-21T14:54:50Z" version="1" changeset="22">
+    <way visible="true" id="5" timestamp="2017-11-29T21:11:52Z" version="1" changeset="6">
         <nd ref="12"/>
         <nd ref="13"/>
         <nd ref="14"/>
         <nd ref="12"/>
     </way>
-    <relation visible="true" id="1" timestamp="2017-09-21T14:54:50Z" version="1" changeset="24">
+    <relation visible="true" id="1" timestamp="2017-11-29T21:11:52Z" version="1" changeset="8">
         <member type="way" ref="4" role="outer"/>
         <member type="way" ref="5" role="inner"/>
         <tag k="building" v="yes"/>
         <tag k="name" v="r1"/>
+        <tag k="type" v="multipolygon"/>
```

Comment:
This is the only change you have to worry about in this file.  The rest are just times and changeset ids.  Relation `type` tag gets added to the OSM like it should.